packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -634,6 +638,40 @@ def async_act_wrapper(self) -> Generator:
         if self._is_done:
             self._log_end()
 
+    def _sync_state(self, app_hash: str) -> None:
+        """
+        Sync the app's state using the given `app_hash`.
+
+        This method is intended for use after syncing local with remote.
+        The fact that we sync up with the blockchain does not mean that we also have the correct application state.
+        The application state is defined by the abci app's developer and also determines the generation of the app hash.
+        When we sync with the blockchain, it means that we no longer lag behind the other agents.
+        However, the application's state should also be updated, which is what this method takes care for.
+        We have chosen a simple application state for the time being,
+        which is a combination of the round count and the times we have reset so far.
+        The round count will be automatically updated by the framework while replaying the missed rounds,
+        however, the reset index needs to be manually updated.
+
+        Tendermint's block sync and state sync are not to be confused with our application's state;
+        they are different methods to sync faster with the blockchain.
+
+        :param app_hash: the app hash from which the state will be updated.
+        """
+        if app_hash == "":
```

Comment:
Can we turn this into a constant? Otherwise this magic value is too implicit imo

Reply:
Done in 0a7eecae6aea2e70f4d0cbb9abc6af4a499c6eaa.